src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -458,8 +494,15 @@ public sealed override IEnumerable<IOperation> Children
         {
             get
             {
-                yield return Target;
-                yield return Value;
+                IOperation current;
+                if ((current = Value) != null)
+                {
+                    yield return current;
+                }
+                if ((current = Target) != null)
```

Comment:
>if ((current = Target) != null) [](start = 16, length = 31)

The assignment target is evaluated first, therefore it should precede ```Value``` in the ```Children``` list.

Reply:
done